deepmd/pt/model/descriptor/se_a_old.py

```
@@ -0,0 +1,713 @@
+)
+
+
+@BaseDescriptor.register("se_e2_a")
+@BaseDescriptor.register("se_a")
+class DescrptSeA(BaseDescriptor, torch.nn.Module):
+    def __init__(
+        self,
+        rcut,
+        rcut_smth,
+        sel,
+        neuron=[25, 50, 100],
```

Ruff

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)